src/test/resources/for-execution/operators.golo

```diff
@@ -77,3 +77,49 @@ function null_guarded = {
   let map = map[]
   return map: get("bogus") orIfNull "n/a"
 }
+
+function polymorphic_number_comparison = {
```

Comment:
you mean a series of `require(a==b, "equals failed")` with no check in the java unit test?